interfaces/santander.py

```diff
@@ -49,11 +49,24 @@ def login(self, driver_cls=webdriver.PhantomJS):
 
 
 		# login
-		passcode_e = driver.find_element_by_id('authentication.PassCode')
-		reg_num_e = driver.find_element_by_id('authentication.ERN')
-		passcode_e.send_keys(self.password)
-		reg_num_e.send_keys(self.reg_num)
-		passcode_e.send_keys(Keys.RETURN)
+
+		for i in range(1, len(self.password) + 1):
+			try:
+				password = driver.find_element_by_id('signPosition' + str(i))
+				password.send_keys(self.password[i-1])
+			except:

```

Comment:
Bare `except` catches `KeyboardInterrupt` and other errors - can you change this to catch a specific error type?